### ISSUE TITLE
Add colorblind friendly color scales

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/OsmAnd#20437]
### Describe the idea (required)

Split off from #20432:
It would be nice if OsmAnd would include more color schemes by default, especially color scales for colorblindness.

As a source for nice colorblind-friendly color scales, please refer to the following sources:
https://davidmathlogic.com/colorblind/
https://www.nceas.ucsb.edu/sites/default/files/2022-06/Colorblind%20Safe%20Color%20Schemes.pdf
https://github.com/sjmgarnier/viridis/
https://colorbrewer2.org/#type=sequential&scheme=BuGn&n=9

### Tell us about the expected behaviour (required)

Colorblind safe color scales

### Tell us about alternatives you've considered (required)

¯\_(ツ)_/¯

**Comment by vshcherb** [comment]
Well I think we need just different scales it's not about color blindness it's also about different data. Even with normal color vision, you can't distinguish some shades cause your data is too dense around average value.

**Comment by scaidermern** [comment]
Yes, there are many different reasons for adding additional color scales. Maybe do this step by step, starting with the easiest tasks. There are already predefined color schemes for colorblindness available, adding them is probably easy :)
